ketchup/main.py

```diff
@@ -33,30 +35,40 @@ def get_date(prompt: str, require_future: bool = False, require_past: bool = Fal
 def run():
     while True:
         try:
-            episodes = int(input(
-                "Hello! How many episodes are in the show you are interested in watching?: "
-            ))
+            episodes = int(
+                input(
+                    "\nHello! How many episodes are in the show you are interested in watching?:\n"
+                )
+            )
         except ValueError:
-            print("Sorry, please only enter a value greater than 0")
+            print("Sorry, please only enter a value greater than 0.")
             continue
 
         if episodes <= 0:
-            print("Sorry, please only enter a value greater than 0")
+            print("Sorry, please only enter a value greater than 0.")
             continue
         else:
             break
 
-    end_date_prompt = "What date would you like to have completed this series?(ex: 15 Aug 2050): "
+    end_date_prompt = (
+        "What date would you like to have completed this series?(Ex: 15 Aug 2050):\n"
+    )
     end_date = get_date(end_date_prompt, require_future=True)
 
-    answer_today = input("Will you start watching today? [Y/n]: ") or "y"
+    answer_today = input("Will you start watching today? [Y/N]:\n") or "y"
```

Comment:
wouldnt this put the user prompt on the new line instead of same line?

Reply:
Personal preference for myself. It just seemed more aesthetically pleasing to me. Not required. Just a possible change.